### PR TITLE
Added nGraph/Transformations/nGarphTests/IECore CODEOWNERS

### PR DESCRIPTION
**Pull request by GlebKazantaev**

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -40,12 +43,14 @@ azure-pipelines.yml  @openvinotoolkit/openvino-admins
...
 /inference-engine/tests_deprecated/  @openvinotoolkit/openvino-ie-tests-maintainers
+/inference-engine/tests/functional/inference_engine/ngraph_reader  @GlebKazantaev @ichuraev
+/inference-engine/tests/functional/inference_engine/transformations  @GlebKazantaev @ilyachur
 
 # MO:
 /model-optimizer/  @openvinotoolkit/openvino-mo-maintainers
 
 # nGraph:
-/ngraph/  @openvinotoolkit/openvino-mo-maintainers
+/ngraph/  @GlebKazantaev @ilyachur
```
Comment (azhogov) [inline CODEOWNERS:53]: the group is added

File: CODEOWNERS [hunk 2]
```diff
@@ -40,12 +43,14 @@ azure-pipelines.yml  @openvinotoolkit/openvino-admins
 # IE Tests:
 /inference-engine/tests/  @openvinotoolkit/openvino-ie-tests-maintainers
 /inference-engine/tests_deprecated/  @openvinotoolkit/openvino-ie-tests-maintainers
+/inference-engine/tests/functional/inference_engine/ngraph_reader  @GlebKazantaev @ichuraev
```
Comment (azhogov) [inline CODEOWNERS:46]: probably should be under the nGraph group below

File: CODEOWNERS [hunk 3]
```diff
@@ -10,8 +10,11 @@ azure-pipelines.yml  @openvinotoolkit/openvino-admins
 # QA Tests:
 /tests/  @openvinotoolkit/openvino-tests-maintainers
 
-# IE in general:
+# IE Core:
 /inference-engine/  @openvinotoolkit/openvino-ie-maintainers
+/inference-engine/src/transformations/  @GlebKazantaev @ichuraev
```
Comment (azhogov) [inline CODEOWNERS:15]: just ask to add into the @openvinotoolkit/openvino-ie-maintainers

File: CODEOWNERS (hunk 1 above)
Comment (ilyachur) [reply CODEOWNERS:53]: It was resolved in the PR: https://github.com/openvinotoolkit/openvino/pull/628

File: CODEOWNERS (hunk 3 above)
Comment (ilyachur) [reply CODEOWNERS:15]: Ok, I will try to join to this group.

Comment (GlebKazantaev) [reply CODEOWNERS:15]: I don't want someone from openvino-ie-maintainers approve changes in this path except me and Ilya.

File: CODEOWNERS (hunk 1 above)
Comment (GlebKazantaev) [reply CODEOWNERS]: Agree

File: CODEOWNERS (hunk 2 above)
Comment (GlebKazantaev) [reply CODEOWNERS]: agree

File: CODEOWNERS (hunk 3 above)
Comment (SDxKeeper) [reply CODEOWNERS:15]: Why don't you create openvino-transformation-maintainers with 2 people?

File: CODEOWNERS [hunk 4]
```diff
@@ -40,6 +43,8 @@ azure-pipelines.yml  @openvinotoolkit/openvino-admins
 # IE Tests:
 /inference-engine/tests/  @openvinotoolkit/openvino-ie-tests-maintainers
 /inference-engine/tests_deprecated/  @openvinotoolkit/openvino-ie-tests-maintainers
+/inference-engine/tests/functional/inference_engine/ngraph_reader  @openvinotoolkit/openvino-ngraph-maintainers
```
Comment (azhogov) [inline CODEOWNERS:46]: should be under # nGraph:

**Pull request merged**